staging/src/k8s.io/kube-scheduler/extender/v1/types.go

```diff
@@ -92,6 +92,10 @@ type ExtenderFilterResult struct {
 	NodeNames *[]string
 	// Filtered out nodes where the pod can't be scheduled and the failure messages
 	FailedNodes FailedNodesMap
+	// Filtered out nodes where the pod can't be scheduled and preemption would
+	// not change anything. The value is the failure message same as FailedNodes.
+	// Nodes specified here takes precedence over FailedNodes.
+	FailedAndUnresolvableNodes FailedNodesMap
```

Comment:
as a serialized API type, this file is strange... it has no json tags, etc... are the scheduler extender APIs being actively expanded?

Reply:
It is not "actively" expanded per se, I think this is the first update in a very long time. This is an old API, it precedes the involvement of most active members in sig scheduling, we are aware that it misses json tags and proper versioning, there is an open issue to fix that:  #88634

Reply:
ok